maestro/agent/helm/templates/maestro-agent.deployment.yaml

```diff
@@ -36,6 +68,13 @@ spec:
           readOnly: true
       serviceAccountName: maestro
       volumes:
+      - name: nginx-config-tmp
+        configMap:
+          name: nginx-config
+      - name: nginx-config
+        emptyDir: {}
+      - name: logs
```

Comment:
We won't need this volume if both access and error logs are sent to null/stderr (not files)